modules/nf-core/ale/meta.yml

```diff
@@ -0,0 +1,57 @@
+---
+name: "ale"
+description: "ALE: assembly likelihood estimator."
+keywords:
+  - reference-independent
+  - assembly
+  - evaluation
+tools:
+  - "ale":
+      description: "ALE is a generic assembly likelihood evaluation framework for assessing the accuracy of genome and metagenome assemblies."
+      documentation: "https://portal.nersc.gov/dna/RD/Adv-Seq/ALE-doc/index.html#document-install"
+      tool_dev_url: "https://github.com/sc932/ALE"
+      doi: "10.1093/bioinformatics/bts723"
+      licence: ["NCSA"]
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'sample1', single_end:false ]`
```

Comment:
Verify that the meta has those fields (it might, but juts make sure)

Reply:
Verified.